21.md

```diff
@@ -0,0 +1,55 @@
+LUD-21: `verify` base spec.
+===============================
+
+`author: reneaaron`
+
+---
+
+Allow `SERVICE`s to check whether an invoice has been paid by contacting the `LN SERVICE`.
+
+A `SERVICE` can verify a payment was made without access to the user's node. Users can give their LNURL or [Lightning Address](https://lightningaddress.com) to a `SERVICE` and allow the `SERVICE` to create *and* verify invoices on their behalf. 
+
+This allows users to re-use existing wallets and `SERVICE`s to facilitate P2P payments without being an intermediary.
+
+## LNURL-verify
+
+An optional field `verify` is added to the `callback` response:
+
+```diff
+{
+  "status": "OK",
+  "routes": [],
+  "pr":     "lnbc10...",
++ "verify": "https://example.com/verify/894e7f7e...",
```

Comment:
```suggestion
  "verify": "https://example.com/verify/894e7f7e...",
```

Extra `+` in there accidentally?

Reply:
This is rendering the diff and showing which line was added. I guess it made more sense in the issue to describe the proposed changes, will remove it :ok_hand: 

![image](https://github.com/lnurl/luds/assets/100827540/963e4c7b-96de-4164-8b55-202d4fd11c57)